### PR TITLE
provider/datadog: make datadog_user verified a computed attribute

### PR DESCRIPTION
**Pull request by phinze**
It's read-only and generates supurious diffs for verified users.

### CODE REVIEW

File: builtin/providers/datadog/resource_datadog_user.go [hunk 1]
```diff
@@ -107,7 +108,7 @@ func resourceDatadogUserRead(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
-	log.Printf("[DEBUG] user: %v", u)
+	log.Printf("[DEBUG] user: %s", spew.Sdump(u))
```
Comment (stack72) [inline builtin/providers/datadog/resource_datadog_user.go]: Can we remove the spew.Dump here? Think this is debug work ;)

Comment (phinze) [inline builtin/providers/datadog/resource_datadog_user.go]: I only left it in as the `#v` output is useless - it's just a bunch of memory addresses. Should we remove the entire line?

**Comment by phinze** [comment]
> Do the docs need updated as well?

Docs already say this is an output-only attribute so this is just fixing up the code to agree with them! 🔧 

https://www.terraform.io/docs/providers/datadog/r/user.html#verified

File: builtin/providers/datadog/resource_datadog_user.go (hunk 1 above)
Comment (stack72) [inline builtin/providers/datadog/resource_datadog_user.go]: I just removed the entire debug line as it's pretty much useless :) <3 for the fixing of the debug though

**Pull request merged**